### PR TITLE
Add Private Service Connect to Looker

### PR DESCRIPTION
**Pull request by efeelaiho**
Adding Add Private Service Connect (PSC) connection to Looker instances 

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
looker:  added `psc_enabled` and `psc_config` to `google_looker_instance` resource
```

### CODE REVIEW

**Comment by efeelaiho** [comment]
@c2thorn any thing I should do here to get this merged?

File: mmv1/templates/terraform/examples/looker_instance_psc.tf.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  public_ip_enabled  = false
+  psc_enabled        = true
+  oauth_config {
+    client_id = "<%= ctx[:vars]["client_id"] %>"
+    client_secret = "<%= ctx[:vars]["client_secret"] %>"
+  }
+  psc_config {
+    allowed_vpcs: ["projects/test-project/global/networks/test"]
+    // update only
+    // service_attachments: [{local_fqdn: "www.local-fqdn.com" target_service_attachment_uri: "projects/my-project/regions/us-east1/serviceAttachments/sa"}]
```
Comment (c2thorn) [inline mmv1/templates/terraform/examples/looker_instance_psc.tf.erb:15]: are these meant to be commented out? Terraform comments use "#"

But if you are trying to make an update test, that should be done in a handwritten file. I'm not sure if one already exists for looker, but you can see our documentation here: https://googlecloudplatform.github.io/magic-modules/develop/test/test/#add-an-update-test

File: mmv1/templates/terraform/examples/looker_instance_psc.tf.erb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+resource "google_looker_instance" "<%= ctx[:primary_resource_id] %>" {
```
Comment (c2thorn) [inline mmv1/templates/terraform/examples/looker_instance_psc.tf.erb:1]: This file does not have a reference in the YAML so it is not doing anything

**Review by c2thorn — CHANGES_REQUESTED** [review]

**Review by c2thorn — CHANGES_REQUESTED** [review]
test is still failing with
```
        Error: Argument or block definition required
        
          on terraform_plugin_test.tf line 14, in resource "google_looker_instance" "looker-instance":
          14:     allowed_vpcs: ["projects/test-project/global/networks/test"]
        
        An argument or block definition is required here. To set an argument, use the
        equals sign "=" to introduce the argument value.
```

File: mmv1/products/looker/Instance.yaml [hunk 3]
```diff
@@ -381,6 +388,55 @@ properties:
...
+              name: localFqdn
+              description: |
+                Fully qualified domain name that will be used in the private DNS record created for the service attachment.
+            - !ruby/object:Api::Type::String
+              name: targetServiceAttachmentUri
+              description: |
+                URI of the service attachment to connect to.
+  # PscConfig Object - End
+  - !ruby/object:Api::Type::Boolean
+    name: pscEnabled
```
Comment (c2thorn) [inline mmv1/products/looker/Instance.yaml:437]: interesting... this is usually in the config field itself.

**Review by c2thorn — APPROVED** [review]

**Pull request merged**